charts/kube-master/templates/reject-proxy-config.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.rejectProxy.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "master.fullname" . }}-reject-proxy
+  namespace: {{ .Release.Namespace }}
```

Comment:
I would remove those references, the can added automatically and just add visual noise to the template

Reply:
I removed namespace references.